biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -206,14 +248,36 @@ subroutine Mortality_Derivative( currentSite, currentCohort, bc_in)
     if (currentCohort%canopy_layer > 1)then 
        ! Include understory logging mortality rates not associated with disturbance
        dndt_logging = (currentCohort%lmort_direct     + &
-                       currentCohort%lmort_collateral + &
-                       currentCohort%lmort_infra)/hlm_freq_day
-       currentCohort%dndt = -1.0_r8 * (cmort+hmort+bmort+frmort+dndt_logging) * currentCohort%n
+            currentCohort%lmort_collateral + &
+            currentCohort%lmort_infra)/hlm_freq_day
+
+       ! this  caps bmort so that daily mortality cannot exceed 0.995
+       if ((cmort+hmort+bmort+frmort+smort+asmort + dndt_logging)*hlm_freq_day &
+            > 0.995_r8)then
```

Comment:
where is the 0.995 coming from?  maybe make a named constant?

Reply:
I'm actually wondering now why I made that 0.995. Should it just be 1?

Reply:
yeah, I don't see why it shouldn't be.  maybe test and if it fails we can try to fix whatever underlying problem might be there?

Reply:
I don't think this cap actually makes sense. I was trying to avoid doubling up with background mortality and size/age mortality. But the way it is written I don't think daily mortality could ever exceed 1 if multiplied by hlm_freq_day. 
Maybe what I need is just bmort+smort or asmort cannot >1. i.e. if the annual rates of just those two terms are exceeding 1 we need to start reducing bmort.